modules/datastore/tests/src/Storage/DatabaseTableTest.php

```diff
@@ -253,7 +254,7 @@ public function testDestroy() {
   /**
    *
    */
-  public function testPrepareDataJsonDecodeNull() {
+  public function testPrepareDataJsonDecodeNULL() {
```

Comment:
Weird casing again. Maybe it stemmed from doing a replace-all on `Null` to `NULL`